File: Driver-App/app/src/main/java/com/seentechs/newtaxidriver/home/interfaces/ServiceListener.java
package com.seentechs.newtaxidriver.home.interfaces;

import com.seentechs.newtaxidriver.common.model.JsonResponse;


public interface ServiceListener {

    void onSuccess(JsonResponse jsonResp, String data);

    void onFailure(JsonResponse jsonResp, String data);


    /*void onSuccessResponse(JsonResponse jsonResp, String data);*/
}

File: User-App/app/src/main/java/com/seentechs/newtaxiusers/appcommon/interfaces/ServiceListener.kt
package com.seentechs.newtaxiusers.appcommon.interfaces

/**
 * @package com.seentechs.newtaxiusers
 * @subpackage interfaces
 * @category ServiceListener
 * @author Seen Technologies
 * 
 */

import com.seentechs.newtaxiusers.appcommon.datamodels.JsonResponse

/*****************************************************************
 * ServiceListener
 */
interface ServiceListener {

    fun onSuccess(jsonResp: JsonResponse, data: String)

    fun onFailure(jsonResp: JsonResponse, data: String)
}
